websites/I/IBL/presence.ts

```diff
@@ -0,0 +1,149 @@
+const presence = new Presence({
+    clientId: '815553000470478850'
+}), timestamp = Math.floor(Date.now() / 1000);
+
+presence.on('UpdateData', async () => {
+    const showTimestamp: boolean = await presence.getSetting('timestamp'),
+    showButtons: boolean = await presence.getSetting('buttons'),

```

Comment:
Use Promise.all and use type generics too

Reply:
@EncryptedDev can you explain this in a little more detail please as this is the only issue i didnt really understand what you mean

Reply:
```suggestion
     const [ showTimestamp,showButtons ] = await Promise.all([
     	presence.getSetting<boolean>('timestamp'),
     	presence.getSetting<boolean>('buttons'),
     ])
```